client/src/components/ChatSideBanner/ChatSideBanner.tsx

```diff
@@ -16,20 +18,48 @@ interface Props {
 const ChatSideBanner = ({ loggedInUser }: Props): JSX.Element => {
   const [search, setSearch] = useState<string>('test');
   const [newChatUser, setNewChatUser] = useState<User | null>(null);
+  const [open, setOpen] = useState<boolean>(false);
+  const [imageSource, setImageSource] = useState<string | null>(null);
+
   const classes = useStyles();
 
-  // React.FormEvent<FormControl & FormControlProps>)
+  useEffect(() => {
+    getProfilePhoto().then((data) => {
+      if (data.success) {
+        setImageSource(data.success.imageURI);
+      }
+    });
+  }, [imageSource]);
+
+  const handleClick = () => {
+    setOpen(true);
+  };
+
   const handleChange = (e: ChangeEvent<HTMLInputElement>, newInputValue: string) => {
     setSearch(newInputValue);
     if (newChatUser) {
       setNewChatUser(null);
     }
   };
 
+  const handleUpload = (files: File[]) => {
+    if (files.length) {
+      uploadProfilePhoto(files[0]).then((data) => {
+        setImageSource(data.success.imageURI);
+      });
+    }
+    setOpen(false);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
```

Comment:
It would be better to implement this menu/dialog on the navbar avatar rather than the ChatSidebar (our project doesn't have a sidebar in mocks, correct?).

Reply:
Yes, this feature was implemented before the navbar was made. Once this PR is approved I will add the components to the new design.